internal/server/statvar/golden/get_statvar_group/earth.json

```diff
@@ -2902,10 +2902,6 @@
         {
           "id": "dc/g/Consumption",
           "specializedEntity": "Consumption"
-        },
```

Comment:
Is this expected?

Reply:
Probably ok... its there under Environment - https://screenshot.googleplex.com/5r4u33QdSVTL2sU - in both autopush and prod